src/reformatters/u_arizona/swann/region_job.py

```diff
@@ -63,6 +72,35 @@ class SWANNRegionJob(RegionJob[SWANNDataVar, SWANNSourceFileCoord]):
     # Be gentle to UA HTTP servers
     download_parallelism: int = 2
 
+    @classmethod
+    def operational_update_jobs(
+        cls,
+        final_store: zarr.abc.store.Store,
+        tmp_store: Path,
+        get_template_fn: Callable[[DatetimeLike], xr.Dataset],
+        append_dim: AppendDim,
+        all_data_vars: Sequence[SWANNDataVar],
+        reformat_job_name: str,
+    ) -> tuple[Sequence[RegionJob[SWANNDataVar, SWANNSourceFileCoord]], xr.Dataset]:
+        append_dim_end = cls._operational_append_dim_end()
+
+        template_ds = get_template_fn(append_dim_end)
+        existing_ds = xr.open_zarr(final_store)
+
+        append_dim_start = cls._operational_append_dim_start(existing_ds)
```

Comment:
this ordering makes a little more sense to me, whatcha think?
```suggestion
        existing_ds = xr.open_zarr(final_store)
        append_dim_start = cls._operational_append_dim_start(existing_ds)
        append_dim_end = cls._operational_append_dim_end()
        template_ds = get_template_fn(append_dim_end)

```